src/ol/style/icon.js

```diff
@@ -260,6 +260,16 @@ ol.style.Icon.prototype.getAnchor = function() {
 
 
 /**
+ * Get the icon color.
+ * @return {ol.Color|string} Color.
+ * @api
+ */
+ol.style.Icon.prototype.getColor = function() {
+  return this.color_;
```

Comment:
Apparently, it's not initialised the same way in the constructor : 
[ol/style/fill.js#L19-L23](https://github.com/openlayers/ol3/blob/48c1d09a95a11e974d05582e5d4c15c5492c0254/src/ol/style/fill.js#L19-L23)
[ol/style/icon.js#L98-L103](https://github.com/openlayers/ol3/blob/48c1d09a95a11e974d05582e5d4c15c5492c0254/src/ol/style/icon.js#L98-L103)